Oracle/PCORNetLoader_ora.sql

```diff
@@ -19,7 +20,6 @@ create or replace PROCEDURE PMN_DROPSQL(sqlstring VARCHAR2) AS
   EXCEPTION
       WHEN OTHERS THEN NULL;
 END PMN_DROPSQL;
-/

```

Comment:
Just read about it - looks like SQL*Plus needs the slashes. (Referring to the ones that you deleted, rather than the ones that you replaced with semicolons.)